config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow]
+# allow FTP traffic after successful HTTP auth
+enabled   = false
+filter    = apache-pass
+banaction = allow-iptables-multiport
+# access log of the website with HTTP auth
+logpath   = /var/log/apache2/access.log
+port      = ftp,ftp-data,ftps,ftps-data
```

Comment:
If this will default to ftp, does it make sense for -ftp to be in the jail name?